What is the point of installing API docs when the library is for internal 
use only (no headers are installed)??

--- origsrc/gnome-chemistry-utils-0.10.3/docs/Makefile.am	2009-01-05 13:09:58.000000000 -0600
+++ src/gnome-chemistry-utils-0.10.3/docs/Makefile.am	2009-01-12 13:04:38.796500000 -0600
@@ -1,3 +1,3 @@
-SUBDIRS = help man reference
+SUBDIRS = help man # reference
 
 MAINTAINERCLEANFILES = Makefile.in
